Typo in java.manifest.template

https://github.com/gramineproject/examples/blob/ebc051be9dad032da789a27387c36db3cbadbc95/openjdk/java.manifest.template#L34

`"file:/usr/lib/jvm/java-11-openjdk-amd64/lib"/` should be `"file:/usr/lib/jvm/java-11-openjdk-amd64/lib/"`, otherwise there would be error message below:
```
javac -d . MultiThreadMain.java
gramine-manifest \
	-Dlog_level=error \
	-Darch_libdir=/lib/x86_64-linux-gnu \
	-Dentrypoint=/usr/lib/jvm/java-11-openjdk-amd64/bin/java \
	java.manifest.template >java.manifest
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/toml/decoder.py", line 512, in loads
    multibackslash)
  File "/usr/local/lib/python3.6/dist-packages/toml/decoder.py", line 778, in load_line
    value, vtype = self.load_value(pair[1], strictly_valid)
  File "/usr/local/lib/python3.6/dist-packages/toml/decoder.py", line 880, in load_value
    return (self.load_array(v), "array")
  File "/usr/local/lib/python3.6/dist-packages/toml/decoder.py", line 1026, in load_array
    nval, ntype = self.load_value(a[i])
  File "/usr/local/lib/python3.6/dist-packages/toml/decoder.py", line 850, in load_value
    "string. Invalid TOML.")
ValueError: Found tokens after a closed string. Invalid TOML.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/gramine-manifest", line 33, in <module>
    main() # pylint: disable=no-value-for-parameter
  File "/usr/lib/python3/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/bin/gramine-manifest", line 29, in main
    manifest = Manifest.from_template(template, define)
  File "/usr/local/lib/python3.6/dist-packages/graminelibos/manifest.py", line 159, in from_template
    return cls(_env.from_string(template).render(**(variables or {})))
  File "/usr/local/lib/python3.6/dist-packages/graminelibos/manifest.py", line 85, in __init__
    manifest = toml.loads(manifest_str)
  File "/usr/local/lib/python3.6/dist-packages/toml/decoder.py", line 514, in loads
    raise TomlDecodeError(str(err), original, pos)
toml.decoder.TomlDecodeError: Found tokens after a closed string. Invalid TOML. (line 28 column 1 char 861)
Makefile:26: recipe for target 'java.manifest' failed
make: *** [java.manifest] Error 1
```

--- a/openjdk/java.manifest.template
+++ b/openjdk/java.manifest.template
@@ -31,7 +31,7 @@ sgx.trusted_files = [
   "file:{{ gramine.runtimedir() }}/",
   "file:{{ arch_libdir }}/",
   "file:/usr/{{ arch_libdir }}/",
-  "file:/usr/lib/jvm/java-11-openjdk-amd64/lib"/,
+  "file:/usr/lib/jvm/java-11-openjdk-amd64/lib/",
   "file:/usr/lib/jvm/java-11-openjdk-amd64/conf/security/java.security",
   "file:/usr/share/java/java-atk-wrapper.jar",
   "file:MultiThreadMain.class",

